### PR TITLE
feat: careers pages

### PR DESCRIPTION
**Pull request by chronark**

### CODE REVIEW

File: apps/web/app/(landing)/careers/[slug]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    title: `${job?.title} | Unkey`,
+    description: job?.description,
+    openGraph: {
+      title: `${job?.title} | Unkey`,
+      description: job?.description,
+      url: `${baseUrl}/careers/${job?.slug}`,
+      siteName: "unkey.dev",
+      images: [
+        {
+          url: `${baseUrl}/og/changelog?title=${job?.title}`,
```
Comment (perkinsjr) [inline apps/web/app/(landing)/careers/[slug]/page.tsx:32]: should this url
be the new job url?

File: apps/web/app/(landing)/careers/page.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    ],
+  },
+  twitter: {
+    title: "Changelog | Unkey",
+    card: "summary_large_image",
+  },
+  icons: {
+    shortcut: "/unkey.png",
+  },
+};
```
Comment (perkinsjr) [inline apps/web/app/(landing)/careers/page.tsx:32]: seems you forgot to update the metadata to the job

**Review by perkinsjr — CHANGES_REQUESTED** [review]

File: apps/web/app/(landing)/careers/[slug]/page.tsx (hunk 1 above)
Comment (chronark) [reply apps/web/app/(landing)/careers/[slug]/page.tsx:32]: yes, what else?

File: apps/web/app/(landing)/careers/page.tsx (hunk 2 above)
Comment (chronark) [reply apps/web/app/(landing)/careers/page.tsx:32]: oops

**Pull request merged**